tools/install_bazel.sh

```diff
@@ -1,13 +1,12 @@
 #!/usr/bin/env bash
 
-BAZEL_VERSION=$1
 if ! command -v sudo >/dev/null; then
     SUDO=
 else
     SUDO=sudo
 fi
 
-echo "Checking if Bazel ${BAZEL_VERSION} needs to be installed..."
+echo "Checking if Bazelisk ${BAZEL_VERSION} needs to be installed..."
```

Comment:
Does it make sense to echo `${BAZEL_VERSION}` if it is not defined?

Reply:
thanks, I have added BAZELISK_VERSION along with renovate auto upgrade.